Source/WebCore/storage/StorageUtilities.cpp

```diff
@@ -93,7 +93,7 @@ String encodeSecurityOriginForFileName(FileSystem::Salt salt, const SecurityOrig
     auto crypto = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
     auto originString = origin.toString().utf8();
     crypto->addBytes(originString.span());
-    crypto->addBytes(salt.data(), salt.size());
+    crypto->addBytes(std::span { salt });
```

Comment:
Do we really need the explicit `std::span { }` here? Maybe we can just pass `salt`?

Reply:
That seems to work, indeed!